web/src/beta/lib/core/StoryPanel/Block/builtin/Layer/Editor/index.tsx

```diff
@@ -67,35 +67,40 @@ const CameraBlockEditor: React.FC<Props> = ({
 
   return (
     <EditorWrapper>
-      <ListField
-        name={t("Buttons List")}
-        items={listItems}
-        addItem={handleItemAdd}
-        removeItem={handleItemRemove}
-        onItemDrop={handleItemMove}
-        selected={selected}
-        onSelect={setSelected}
-        atLeastOneItem
-      />
-      <FieldGroup disabled={!editorProperties}>
-        <TextField
-          name={editorProperties?.title?.title}
-          description={editorProperties?.title?.description}
-          value={editorProperties?.title?.value}
-          onChange={value => debounceOnUpdate(selected, "title", "string", value)}
-        />
-        <ColorField
-          name={editorProperties?.color?.title}
-          description={editorProperties?.color?.description}
-          value={editorProperties?.color?.value}
-          onChange={value => debounceOnUpdate(selected, "color", "string", value)}
-        />
-        <ColorField
-          name={editorProperties?.bgColor?.title}
-          description={editorProperties?.bgColor?.description}
-          value={editorProperties?.bgColor?.value}
-          onChange={value => debounceOnUpdate(selected, "bgColor", "string", value)}
+      <GroupWrapper>
+        <ListField
+          name={t("Buttons List")}
+          items={listItems}
+          addItem={handleItemAdd}
+          removeItem={handleItemRemove}
+          onItemDrop={handleItemMove}
+          selected={selected}
+          onSelect={setSelected}
+          atLeastOneItem
         />
+        <FieldGroup disabled={!editorProperties}>
+          <TextField
+            name={t("Button Title")}
```

Comment:
Does this property not have a title (`editorProperties?.title?.title`)? If it doesn't, it should, so its better to add to the manifest than make this a unique case on the frontend

Reply:
there is the title so I added it